ReactWindows/ReactNative/ReactNative.csproj

```diff
@@ -1,386 +1,394 @@
+    <Compile Include="UIManager\UIViewOperationQueue.cs" />
+    <Compile Include="UIManager\ViewAtIndex.cs" />
+    <Compile Include="UIManager\ViewParentManager.cs" />
+    <Compile Include="UIManager\PanelViewParentManager.cs" />
+    <Compile Include="UIManager\ViewManager.cs" />
+    <Compile Include="UIManager\ViewManagerRegistry.cs" />
+    <Compile Include="Views\Scroll\ReactScrollViewManager.cs" />
+    <Compile Include="Views\TextInput\ReactTextBoxProperties.cs" />
+    <Compile Include="Views\TextInput\ReactTextInputShadowNode.cs" />
+    <Compile Include="Views\Switch\ReactSwitchManager.cs" />
+    <Compile Include="Views\Switch\ReactSwitchShadowNode.cs" />
+    <Compile Include="Views\ViewManagersPropertyCache.cs" />
+    <Compile Include="UIManager\ViewProperties.cs" />
+    <Compile Include="Views\Text\ReactRawTextManager.cs" />
+    <Compile Include="Views\Text\ReactTextShadowNode.cs" />
+    <Compile Include="Views\Text\ReactTextViewManager.cs" />
+    <Compile Include="Views\Text\ReactVirtualTextViewManager.cs" />
+    <Compile Include="Views\TextInput\ReactTextChangedEvent.cs" />
+    <Compile Include="Views\TextInput\ReactTextInputBlurEvent.cs" />
+    <Compile Include="Views\TextInput\ReactTextInputFocusEvent.cs" />
+    <Compile Include="Views\View\ReactViewManager.cs" />
+    <Compile Include="Views\TextInput\ReactTextInputManager.cs" />
+    <Compile Include="Modules\Toast\ToastHelper.cs" />
+    <Compile Include="Modules\Toast\ToastModule.cs" />
+    <EmbeddedResource Include="Properties\ReactNative.rd.xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="Bridge\JavaScriptModuleBase.Generated.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>JavaScriptModuleBase.Generated.cs</LastGenOutput>
+    </Content>
+  </ItemGroup>
+  <ItemGroup>
+    <Service Include="{508349B6-6B84-4DF5-91F0-309BEEBAD82D}" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Facebook.CSSLayout">
+      <HintPath>..\References\Facebook.CSSLayout.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+  <ItemGroup>
+    <Page Include="DevSupport\DevOptionDialog.xaml">
+      <SubType>Designer</SubType>
+      <Generator>MSBuild:Compile</Generator>
+    </Page>
+    <Page Include="DevSupport\ProgressDialog.xaml">
+      <SubType>Designer</SubType>
+      <Generator>MSBuild:Compile</Generator>
+    </Page>
+    <Page Include="DevSupport\RedBoxDialog.xaml">
+      <SubType>Designer</SubType>
+      <Generator>MSBuild:Compile</Generator>
+    </Page>
+  </ItemGroup>
+  <ItemGroup>
+    <SDKReference Include="WindowsDesktop, Version=10.0.10586.0">
+      <Name>Windows Desktop Extensions for the UWP</Name>
+    </SDKReference>
+    <SDKReference Include="WindowsMobile, Version=10.0.10586.0">
+      <Name>Windows Mobile Extensions for the UWP</Name>
+    </SDKReference>
+  </ItemGroup>
+  <PropertyGroup Condition=" '$(VisualStudioVersion)' == '' or '$(VisualStudioVersion)' &lt; '14.0' ">

```

Comment:
I don't necessarily like the addition of these dependencies, we need to think about this a bit more...